tests/core/fs/test_fs_sync.py

```diff
@@ -79,7 +79,8 @@
 @pytest.mark.trio
 @pytest.mark.parametrize("type", ["file", "folder"])
 async def test_new_empty_entry(type, running_backend, alice_user_fs, alice2_user_fs):
-    wid = await create_shared_workspace("w", alice_user_fs, alice2_user_fs)
+    with freeze_time("2000-01-01"):
+        wid = await create_shared_workspace("w", alice_user_fs, alice2_user_fs)
```

Comment:
Why is that necessary?

Reply:
Because otherwise shared workspace is created with current time, but after that we do a `workspace.sync()` with time frozen at 2000-01-03.
So the timestamp consistency test failed given there is no role certificate available at the time we upload the data